adapters/tappx/tappx.go

```diff
@@ -5,8 +5,9 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"regexp"
 	"strconv"
-	"strings"
+	//"strings"
```

Comment:
Can you please remove this import if it is no longer needed?

Reply:
Removed the import in commit 793b2b4
Updated the docs in pull request: https://github.com/prebid/prebid.github.io/pull/3520